# исключение


# try:
#     x = 1
# except ZeroDivisionError:
#     print("Ошибка: деление на ноль невозможно")

# try:
#     num = int(input("введите число: "))
#     result = 10 / num
# except (ZeroDivisionError, ValueError): #обрабатывает ошибку
#     print("Ошибка: деление на ноль невозможно \n введите коректно число")
# else:
#     print(result)
# finally: #будет работать всегда!
#     print("этот код выполнится всегда")



# функция 

